project/Dependencies.scala

```diff
@@ -22,9 +22,9 @@ object Dependencies {
 
   val jGit = "org.eclipse.jgit" % "org.eclipse.jgit" % "6.2.0.202206071550-r"
 
-  val remoteRobot = "com.intellij.remoterobot" % "remote-robot" % "0.11.13"
+  val remoteRobot = "com.intellij.remoterobot" % "remote-robot" % "0.11.15"
```

Comment:
Huh I thought I've bumped that already 🤔

Reply:
Yeah... I'm checking v0.36.1:

![Uploading image.png…]()

Reply:
Lemme see what happened in the meantime...

Reply:
Whoops this seems to have been reverted with @wiacekm's commit ca4462f0d83fd237042009cef3d9dc74b7ce03a6.

Michał pls check if there weren't any other undesired changes... `git log -p project/`

Reply:
And @tpasternak if you could launch a release for the current state of `master`